### ISSUE TITLE
Show todos in calendar

### ISSUE THREAD

**Issue by dholm** [dalyons/redmine-todos-scrum-plugin#16]
Todo items should show up in the calendar so that there is one place you can go to keep track of everything that needs to be done before a certain date.

**Comment by dalyons** [comment]
good idea. I will see if there is plugin hooks for the redmine calendar when i get some time.

**Comment by cerupcat** [comment]
I second this. :)
